service/src/main/java/com/ge/predix/acs/monitoring/AcsDBHealthIndicator.java

```diff
@@ -55,6 +61,12 @@ public Health health() {
         if (errorCode != 0) {
             return Health.status(ACS_DB_OUT_OF_SERVICE).withDetail("Error Code for ACS DB", errorCode).build();
         }
+
+        if (null != this.migrationManager && !this.migrationManager.isMigrationComplete()) {
+            return Health.status(ACS_DB_MIGRATION_INCOMPLETE).withDetail("Migration not complete.", FAILED_CHECK)
+                    .build();
+        }
```

Comment:
@sanjeevchopra,
For ACS health indicator to return ACS_DB_MIGRATION_INCOMPLETE custom status, we need to configure a default implementation of health aggregator using the management.health.status.order configuration property in application.properties.  Bharath and I committed that change.  Could you please review and squash the commits.